### PR TITLE
add RoIAlign for faster RCNN model

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

File: fluid/faster_rcnn/models/model_builder.py [hunk 1]
```diff
@@ -196,12 +196,21 @@ def fast_rcnn_heads(self, roi_input):
...
-            pooled_height=14,
-            pooled_width=14,
-            spatial_scale=0.0625)
+        if cfg.roi_func == 'RoIPool':
+            pool = fluid.layers.roi_pool(
+                input=roi_input,
+                rois=pool_rois,
+                pooled_height=14,
+                pooled_width=14,
+                spatial_scale=0.0625)
```
Comment (xingyuanbu) [inline fluid/faster_rcnn/models/model_builder.py]: pooled_width and spatial_scale should leave as a config in config.py
For example, pooled_height may take 7 or 14

Comment (jerrywgz) [reply fluid/faster_rcnn/models/model_builder.py]: Configs are added

File: fluid/faster_rcnn/config.py [hunk 2]
```diff
@@ -150,6 +150,12 @@
 # sampling ratio for roi align
 _C.sampling_ratio = 0
 
+# pooled width and pooled height 
+_C.roi_resolution = 14
+
+# spatial scale 
+_C.spatial_scale = 0.0625
```
Comment (xingyuanbu) [inline fluid/faster_rcnn/config.py]: 0.0625 is meanless, 1/16 would be better~

Comment (jerrywgz) [reply fluid/faster_rcnn/config.py]: Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**